web/viewer.css

```diff
@@ -1899,3 +1905,13 @@ html[dir="rtl"] #documentPropertiesOverlay .row > * {
     display: none;
   }
 }
+
+.dir-rev {
+  transform: scaleX(-1);
```

Comment:
Now that the RTL issue is fixed, if you rebase this PR can't we remove this and the one below then, since you indicated it was a work-around?

Reply:
We have three possible solutions : 
-Get back the two buttons "Left to right" and "right to left"
-Use getcomputedstyle to toggle the document direction.
-Leave it like that to avoid using getcomputedstyle